query/query_runner.pl

```diff
@@ -0,0 +1,40 @@
+
+main(Query,Filename):-
+	consult(Filename),
+	run_query(Query).
+
+run_query(Query):-
+	findall(Result, (atom_to_term(Query, Term, Result), Term), ResultSet),
+	prettyWriteResultSet(ResultSet).
+
+prettyWriteResultSet([]):-
+	writeln('false.').
```

Comment:
Some Prolog implementations use `yes` and `no` instead of `true` and `false`. 

I think those terms are better since they try to emphasize that the meaning of true is different in logic programming than in other paradigms.